backend/graph/schema.graphqls

```diff
@@ -257,6 +265,11 @@ type QueryRoot{
     id: ID!
   ): Section
 
+  search(
+    queryString: String!
+    page: Int!
+  ): SearchResult!
+
```

Comment:
Fyi: This is the generally accepted way of doing pagination in GQL:
https://graphql.org/learn/pagination/

The idea is that the API doesn't control what a page is.